test/python/transpiler/test_basis_translator.py

```diff
@@ -928,22 +928,19 @@ def test_cx_bell_to_ecr(self):
         in_dag = circuit_to_dag(bell)
         out_dag = BasisTranslator(std_eqlib, ["ecr", "u"]).run(in_dag)
 
-        #         ┌────────────┐   ┌─────────────┐┌──────┐┌──────────┐
-        # q_0: ───┤ U(π/2,0,π) ├───┤ U(0,0,-π/2) ├┤0     ├┤ U(π,0,π) ├
-        #      ┌──┴────────────┴──┐└─────────────┘│  Ecr │└──────────┘
-        # q_1: ┤ U(-π/2,-π/2,π/2) ├───────────────┤1     ├────────────
-        #      └──────────────────┘               └──────┘
-
         qr = QuantumRegister(2, "q")
         expected = QuantumCircuit(2)
         expected.u(pi / 2, 0, pi, qr[0])
         expected.u(0, 0, -pi / 2, qr[0])
-        expected.u(-pi / 2, -pi / 2, pi / 2, qr[1])
+        expected.u(pi, 0, 0, qr[0])
+        expected.u(pi / 2, -pi / 2, pi / 2, qr[1])
         expected.ecr(0, 1)
-        expected.u(pi, 0, pi, qr[0])
         expected_dag = circuit_to_dag(expected)
```

Comment:
My question from the prior review! It got lost before - I must have forgotten to save the comment.

Do we know why this changes?  We haven't added any rules that go _to_ `IGate`, so I'd not have expected the basis-translation path to change.

Reply:
TBH, I'm not sure. I didn't really think to check. I just saw the output circuit was equivalent and updated the test. I'll do some debugging locally because the qpy tests are failing in a weird way now too so this is blocked on that anyway.